templates/web/surrey/tracking_code.html

```diff
@@ -0,0 +1,20 @@
+[% IF c.config.BASE_URL == "https://www.fixmystreet.com" %]
+
+<!-- Google Tag Manager -->
+<script nonce="[% csp_nonce %]">
+if (document.cookie.match('(^|;)\\s*google_stats\\s*=\\s*true')) {
```

Comment:
Where does this `google_stats` line come from? That cookie doesn't appear to ever be set anywhere so this will always return false.